### ISSUE TITLE
Bookmarklet

### ISSUE THREAD

**Issue by omahlama** [paulirish/memory-stats.js#6]
I created a bookmarket to add this script to any page without making modifications. I think this could be included in the documentation, as others may find this useful.

Original code:
http://jsbin.com/tibalayose/2/edit

Bookmarklet:

```
javascript:(function()%7Bvar%20module%3Bvar%20script%3Ddocument.createElement(%27script%27)%3Bscript.onload%3Dfunction()%7Bvar%20stats%3Dnew%20MemoryStats()%3Bstats.domElement.style.position%3D%27fixed%27%3Bstats.domElement.style.right%3D%270px%27%3Bstats.domElement.style.bottom%3D%270px%27%3Bdocument.body.appendChild(stats.domElement)%3BrequestAnimationFrame(function%20rAFloop()%7Bstats.update()%3BrequestAnimationFrame(rAFloop)%3B%7D)%3B%7D%3Bscript.src%3D"https://rawgit.com/paulirish/memory-stats.js/master/memory-stats.js"%3Bdocument.getElementsByTagName(%27head%27)%5B0%5D.appendChild(script)%3B%7D)()
```

**Comment by paulirish** [comment]
Can't really maintain that to keep up with the source base. 

If there is some service that bookmarklet's a js file in github.. then.. i guess that would work.

**Comment by nsisodiya** [comment]
we can have a small grunt code to create dynamic bookmarklet , that will be easy using grunt..

**Comment by omahlama** [comment]
Good idea @nsisodiya, I'll try that and make a pull request.

**Comment by omahlama** [comment]
I create a pull request https://github.com/paulirish/memory-stats.js/pull/7 , I didn't create a dynamic bookmarklet, instead modified it to just load a script from github.

**Comment by paulirish** [comment]
fixed by #7

**Comment by awerlang** [comment]
There's a leaked close brace on the last line of https://github.com/paulirish/memory-stats.js/blob/master/bookmarklet.js
